### ISSUE TITLE
error: the target 'SwiftSyntax' in product 'SwiftSyntax' contains unsafe build flags

### ISSUE THREAD

**Issue by philviso-reverb** [Flinesoft/BartyCrouch#249]
Our app has a bootstrap script we run to set up new development environments. The script installs homebrew, mint, and then builds bartycrouch through mint. Recently this script started failing to build bartycrouch. Something changed with how `SwiftSyntax` includes unsafe flags in its `Package.swift` file, and it seems like it's including them in cases where it didn't used to. When I pin the `SwiftSyntax ` version to exact version "0.50500.0" instead of from version, the build works again. Not sure what the best longterm fix is here though.

Here's a commit of what I mean (it's being changed from revision because I tried that as well - also worked). I can put up a PR for this if needed.

https://github.com/philviso-reverb/BartyCrouch/commit/a2ecc766ba981ed0fff3f1f591bd39a08b1a7207

## Expected Behavior

bartycrouch builds successfully

## Actual Behavior

bartycrouch build fails with this error...

```
'SwiftSyntax' 
error: the target 'SwiftSyntax' in product 'SwiftSyntax' contains unsafe build flags
```

More information about that error:

https://forums.swift.org/t/question-about-contains-unsafe-build-flags-in-spm/47677

## Steps to Reproduce the Problem

1. Install [mint](https://github.com/yonaskolb/Mint)
2. Create a Mintfile with these contents

```
Flinesoft/BartyCrouch@4.9.0
nicklockwood/SwiftFormat@0.48.18
realm/SwiftLint@0.45.0
```

(Not sure if that other stuff matters, but figured I'd throw it in here just in case it does)

  3.  run `mint bootstrap`

## Specifications

  - Version: 4.9.0
  - Platform: MacOS 12.2.1
  - IDE Version: Xcode 13.2.1

**Comment by philviso-reverb** [comment]
@Jeehut Thanks for the update!

**Comment by FlineDevPublic** [comment]
@philviso-reverb I'm currently working on the new release in public during my live stream. Join me if you're interested: https://www.twitch.tv/Jeehut

**Comment by FlineDevPublic** [comment]
@philviso-reverb I just [released version 4.10.0](https://github.com/Flinesoft/BartyCrouch/releases/tag/4.10.0) with a fix. Should be live once https://github.com/Homebrew/homebrew-core/pull/97188 is merged. 🎉 

---
_This comment was written during my regular **Open Source live stream on [Twitch](https://www.twitch.tv/jeehut)**. Follow me there to support my work!_

**Comment by philviso-reverb** [comment]
@FlineDevPublic Thanks! (Sorry for the late reply, I totally missed your comment)
